packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1001,7 +1001,10 @@ export default class Meetings extends WebexPlugin {
   fetchUserPreferredWebexSite() {
     return this.request.getMeetingPreferences().then((res) => {
       if (res) {
-        this.preferredWebexSite = MeetingsUtil.parseDefaultSiteFromMeetingPreferences(res);
+        const preferredWebexSite = MeetingsUtil.parseDefaultSiteFromMeetingPreferences(res);
```

Comment:
Why did we change this? I believe there is no difference between the direct and indirect assignments in terms of performance.

Reply:
main reason i did it was because on line 1007 i do
```
this.webex.internal.services._getCatalog().addAllowedDomains([preferredWebexSite]
```
and i didn't want to have to call `MeetingsUtil.parseDefaultSiteFromMeetingPreferences(res);` again